etl/steps/data/meadow/happiness/2023-03-20/happiness.meta.yml

```diff
@@ -0,0 +1,37 @@
+dataset:
+  title: World Happiness Report
```

Comment:
I would probably use the format `Name (year, source)`. However, I am unsure about the source name, seems like dataset name = source name. Perhaps we can go with "World Happiness Report (2023)" then.